report_xlsx/README.rst

```diff
@@ -0,0 +1,90 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
png

Reply:
> There's already another PR for migrating this module in #248 by @ovnicraft. You should both coordinate. This ones seems to be more polite, although:
> 
> * Why do you remove translations?
> * Why do you duplicate commits? Squash all migration commits in one or at least avoid duplicated commits.
> 
> About the error in Travis, there was an incorrect PG version definition. If you rebase over current 12.0 branch, you will get that correct.
Thanks for review @pedrobaeza. Remove temporaly traslantions for test modules with new nomenclature of i18n (model_terms in some fields).  The next commit will have traslations back.
@elicoidal Thanks for you review. I change now.